scheduler/src/main/java/org/cloudfoundry/autoscaler/scheduler/quartz/AppScalingScheduleEndJob.java

```diff
@@ -43,12 +43,12 @@ private void deleteActiveSchedule(JobExecutionContext jobExecutionContext) throw
 		long scheduleId = jobDataMap.getLong(ScheduleJobHelper.SCHEDULE_ID);
 		long startJobIdentifier = jobDataMap.getLong(ScheduleJobHelper.START_JOB_IDENTIFIER);
 
-		boolean activeScheduleTableTaskDone = jobDataMap.getBoolean(ScheduleJobHelper.ACTIVE_SCHEDULE_TABLE_TASK_DONE);
+		boolean activeScheduleTableDeleteTaskDone = jobDataMap.getBoolean(ScheduleJobHelper.ACTIVE_SCHEDULE_TABLE_DELETE_TASK_DONE);
 
-		if (!activeScheduleTableTaskDone) {
+		if (!activeScheduleTableDeleteTaskDone) {
 			try {
 				activeScheduleDao.delete(scheduleId, startJobIdentifier);
-				jobDataMap.put(ScheduleJobHelper.ACTIVE_SCHEDULE_TABLE_TASK_DONE, true);
+				jobDataMap.put(ScheduleJobHelper.ACTIVE_SCHEDULE_TABLE_DELETE_TASK_DONE, true);
```

Comment:
Where is the test to verify this is being set?
Why do we bother guarding this? Why not just always attempt the delete?